docs/how_it_works.rst

```diff
@@ -1,9 +1,34 @@
 How It Works
 ============
 
+General bare metal automation with PXE
+--------------------------------------
 
-Discovery
----------
+As we shared in :doc:`introduction`, the RackHD system leverages standard open source tools
+and protocols. `PXE`_ is the most important, originally created as a specification by Intel
```

Comment:
restructured text makes that a link - either to a hyperlink inside this doc or (as I'm using it) to a title or subheading used in the same doc. I couldn't find a way to reference a title or subheading from another doc :-( Hence my merging of the glossary content and this file